### PR TITLE
Pre-release suggestions

### PR DESCRIPTION
**Pull request by alexanderedge**
Hey Emilia, here are some suggestions before we send to candidates. Let me know what you think!

### CODE REVIEW

File: BCTestTask/Views/MainView.swift [hunk 1]
```diff
@@ -8,8 +8,11 @@
 import SwiftUI
 
 struct MainView {
-    @Environment(\.contentInsets) private var contentInset
     @State private var pinnedContentHeight: CGFloat = 0
+
+    func buttonAction() {
+        print("Load data")
+    }
```
Comment (alexanderedge) [inline BCTestTask/Views/MainView.swift:15]: I moved this function into the struct as it isn't part of `View` conformance

File: BCTestTask.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -312,7 +308,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 17.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```
Comment (alexanderedge) [inline BCTestTask.xcodeproj/project.pbxproj:311]: I didn't have any simulators to run the app on 17.2! Perhaps more of a _me_ problem...

File: BCTestTask.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -189,10 +186,9 @@
 				A5354B9A2B86568800FE684B /* MainView.swift in Sources */,
 				A56168332B8CE80100E33784 /* ButtonTitleView.swift in Sources */,
 				A56168312B8CE77400E33784 /* DesignConstants.swift in Sources */,
-				A5796A7F2B8B7F9B006C6B81 /* ContentInsets.swift in Sources */,
 				A56168352B8CE85E00E33784 /* ComfortableReadingWidth.swift in Sources */,
-				A5796A832B8B8BEA006C6B81 /* ContentEdgeInset.swift in Sources */,
-				A561682B2B8CE68A00E33784 /* CapsuledButtonStyle.swift in Sources */,
+				A5796A832B8B8BEA006C6B81 /* BoundsChangeObserver.swift in Sources */,
+				A561682B2B8CE68A00E33784 /* CapsuleButtonStyle.swift in Sources */,
```
Comment (alexanderedge) [inline BCTestTask.xcodeproj/project.pbxproj:191]: Some renaming to match the actual type names

File: BCTestTask.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -189,10 +186,9 @@
 				A5354B9A2B86568800FE684B /* MainView.swift in Sources */,
 				A56168332B8CE80100E33784 /* ButtonTitleView.swift in Sources */,
 				A56168312B8CE77400E33784 /* DesignConstants.swift in Sources */,
-				A5796A7F2B8B7F9B006C6B81 /* ContentInsets.swift in Sources */,
```
Comment (alexanderedge) [inline BCTestTask.xcodeproj/project.pbxproj:192]: I didn't feel `ContentInsets` added any value in this sample project

File: BCTestTask/Views/MainView.swift [hunk 5]
```diff
@@ -56,7 +54,7 @@ extension MainView: View {
     }
     
     var content: some View {
-        Text("Load Content Here")
+        Text("Display Content Here")
```
Comment (alexanderedge) [inline BCTestTask/Views/MainView.swift:57]: This might be a bit clearer to the candidate

File: BCTestTask.xcodeproj/project.pbxproj (hunk 4 above)
Comment (totemi) [reply BCTestTask.xcodeproj/project.pbxproj:192]: Sure, makes sense

**Review by totemi — APPROVED** [review]
Awesome, thank you so much for this!

**Pull request merged**